hash/adam_flop.xml

```diff
@@ -4391,7 +4391,7 @@ license:CC0
 	</software>
 
 	<software name="gloryba" cloneof="gloryb">
-		<description>Glory B! (Alt Disk 1)</description>
+		<description>Glory B! (alt disk 1)</description>
```

Comment:
“Disk” should probably still be uppercase here, as it’s part of the disk label, which presumably does use title case (see the `part_id` on the first disk).